src/Storages/ObjectStorage/StorageObjectStorageSettings.h

```diff
@@ -67,4 +96,8 @@ struct StorageObjectStorageSettings
 
 using StorageObjectStorageSettingsPtr = std::shared_ptr<StorageObjectStorageSettings>;
 
+#define LIST_OF_STORAGE_OBJECT_STORAGE_SETTINGS(M, ALIAS) \
+    STORAGE_OBJECT_STORAGE_RELATED_SETTINGS(M, ALIAS) \
+    LIST_OF_ALL_FORMAT_SETTINGS(M, ALIAS)
```

Comment:
Out of curiosity. Why do we need to support all format settings on object storage tables?